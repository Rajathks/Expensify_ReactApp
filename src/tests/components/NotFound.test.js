import React from "react";
import {shallow} from "enzyme";
import NotFoundPage from "../../components/NotFoundPage";

test("Not Found Component Test : ", () =>{
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});