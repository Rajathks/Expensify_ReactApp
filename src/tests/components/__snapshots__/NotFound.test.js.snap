// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Component Test :  1`] = `
<div>
  404!! 
  <Link
    to="/"
  >
    Home Page
  </Link>
</div>
`;
